en/slides/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Subtitles on your presentation deck
+---
+
+# Subtitles on your presentation deck
+
+We want to make it a fun event for both the Japanese audience and those visiting Japan from overseas.
+So here are some of the things we will do this year:
+
+- CFP in English and Japanese
+- Speakers are decided by votes
+- All slides in English &lt; you are here
+- Separete tracks for presentations in English and Japanese (with a few exceptions possible)
+- Professional interpreters (English/Japanese bidirection) in two main halls
+- Travel support for speakers with most votes
+- Organize groups to hang out at bars after the conference if you're up for it
+
+As we've mentioned, all presentation deck will be in English.
+However, given that the majority of the audience are Japanese speakers, we normally put "subtitles" to the slides every year.
+
+- [Building a Unified Data Pipline in Spark](http://www.slideshare.net/scalaconfjp/building-a-unified-data-pipline-in-spark)
+- [The Evolution of Scala](http://www.slideshare.net/scalaconfjp/the-evolution-of-scala-scala)
+- [Sample Deck (Keynote 09)](/img/sub-samples/sub-sample.key)
+- [Sample Deck (PDF)](/img/sub-samples/sub-sample.pdf)
+- [Sample Deck (PowerPoint)](/img/sub-samples/sub-sample.pptx)
+
+Like the above sample decks, please create a black text box to fit **2 lines** in **36px font** (27pt on Windows, 36pt on Mac) at the bottom each slides.
+The ScalaMatsuri staff will provide the translation, unless you speak both languages like Chris Birchall.
+
+## Submit your slides
+
+Please submit your slide decks by Dec 15. (It's ok to be late a bit)
+
+ScalaMatsuri staff will provide the translation, and also check for Code of Conduct compliance.
+Since the translation will be done based on text, if you could submit a text file with all the texts, it will be appreciated like a lot!
+We are planning to return the slide deck around January 16. If you have any changes by then, feel free to submit the new deck along with the change description such as "changed the text on page 3." Any changes after January 16 will be at your own risk.
```

Comment:
"We are planning to return the slide deck around January 16, *and also submit it to the interpreters*."